code/dashboard-2/src/Constants.ts

```diff
@@ -721,3 +723,15 @@ export const JOB_QUERY_VALIDATION_SCHEMA = yup.object({
     .required('GPU usage is required')
 })
 
+export const JOB_QUERY_EXPORT_VALIDATION_SCHEMA = yup.object({
+  format: yup.string().required('A format must be selected'),
+  fields: yup.array().of(yup.string()).min(1, 'At least one field must be selected'),
+})
+
+export const JOB_QUERY_EXPORT_FORMATS: ExportFormat[] = [
+  {label: 'CSV with field header', value: 'csvnamed', mimeType: 'text/csv', fileExtension: 'csv'},
```

Comment:
Should be 'CSV with embedded field names'; "header" suggests something external but the point here is that it is not.